### PR TITLE
feat: [CET-1882]: Added icons for CET Nav2 Tiles

### PR DESCRIPTION
**Pull request by abeg777**
You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- StyleLint: `retrigger stylelint`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- ImageSnapshot `retrigger ImageSnapshot`

### CODE REVIEW

File: packages/icons/src/AgentSettings.svg [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<svg xmlns="http://www.w3.org/2000/svg" version="1.0" width="1176.000000pt" height="1176.000000pt" viewBox="0 0 1176.000000 1176.000000" preserveAspectRatio="xMidYMid meet">
```
Comment (knagurski) [inline packages/icons/src/AgentSettings.svg:1]: Can you remove the `width` and `height` attributes and run these files through an optimiser. I like myself https://imageoptim.com/mac

File: packages/icons/src/agent-settings.svg
Comment (knagurski) [inline packages/icons/src/agent-settings.svg:1]: These icons all seem to be blank and they still have a width and height. I'd revert your latest commit, remove the `width`, `height` (and probably `preserveAspectRatio`) attributes and then run through an image optimiser; a quick run through ImageOptim shows that around 16% of the file size can be saved.

File: packages/icons/src/AgentSettings.svg (hunk 1 above)
Comment (abeg777) [reply packages/icons/src/AgentSettings.svg]: Optimized images and removed fields as suggested

**Review by knagurski — DISMISSED** [review]
Looks good 👍

**Review by jsentongo — APPROVED** [review]

**Review by knagurski — APPROVED** [review]
👍🏻

**Pull request merged**